knowledge-base/window-center-programmatically.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How To Center Window Programmatically
+description: Learn how to center a Telerik Window programmatically by using the Top and Left parameters.
+type: how-to
+page_title: How To Center Window Programmatically
+slug: window-kb-center-programmatically
+tags: window, center, position, blazor,
+res_type: kb
+ticketid:
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Window for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How can I programmatically center a Telerik Window?
+* How do I reset a Telerik Window to its default position?
+* How can I position a Telerik Window in the center of the viewport?
+* How do I dynamically adjust the Telerik Window position?
+
+## Solution
+
+To center a Telerik Window programmatically, follow these steps:
+
+1. Use [`Top` and `Left` parameters](slug:components/window/position#top-and-left) – These parameters define the Window position on the screen.
```

Comment:
This step is not actionable, especially if the customer wants the Window to be initially centered. For example, explain:
* That the parameters must use two-way binding.
* What should be their default values.

This may also help you to make the next step simpler, because the reader will already be familiar with the `string.Empty` value.